### PR TITLE
Add button to PR diff file view for expanding/collapsing all files

### PR DESCRIPTION
**Pull request by antiguab**
Helps with really big PRs where I only need to see one or two files out of 20.

### CODE REVIEW

**Comment by hkdobrev** [comment]
Background:

- Initially we had such buttons #51
- GitHub changed layouts and they didn't work everywhere #85
- We thought it would be better to remove the buttons as they weren't that useful #93 

Said that, they could of course be considered again if people want them. I personal don't find much gain in them as I often use the file switcher at the top.w

**Comment by antiguab** [comment]
I found it useful, but I can see it the other way too!

**Comment by sindresorhus** [comment]
@antiguab I'm willing to add it again if you can commit to maintaining it?

File: extension/collapse-expand-all-button.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+window.addCollapseExpandAllButton = () => {
+	// Buttons already added
+	if ($('.collapse-expand-btn').length) {
+		return;
+	}
+
+	const collapseExpandButton = $(`<a class="btn btn-sm BtnGroup-item tooltipped tooltipped-s collapse-expand-btn" aria-label="Collapse All Files"></a>`).html('Collapse All');
+	collapseExpandButton.data('state', 'collapseAll');
+
+	$('.pr-review-tools').prepend($(`<div class="diffbar-item"></div>`).append(collapseExpandButton));
```
Comment (sindresorhus) [inline extension/collapse-expand-all-button.js]: No reason to use template literals here.

File: extension/collapse-expand-all-button.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		$(`div[id^='diff-']`).each((index, file) => {
+			if (state === 'collapseAll') {
+				// collapse all
+				if ($(file).hasClass('refined-github-minimized')) {
+					return;
+				}
+
+				$(file).toggleClass('refined-github-minimized');
+			} else {
+			// expand all
```
Comment (sindresorhus) [inline extension/collapse-expand-all-button.js]: Wrong indent

**Comment by sindresorhus** [comment]
> I'm willing to add it again if you can commit to maintaining it?

Silence is answer enough. Closing.

**Pull request closed without merging**